docs/reference/query-languages/esql/_snippets/functions/appendix/count_distinct.md

```diff
@@ -0,0 +1,24 @@
+% This is generated by ESQL's AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.
```

Comment:
Is this ../README.md right? (Unless it's already in main, I can't see it)

Reply:
This readme did exist in main, but moved. The new location is reference/query-languages/esql/README.md. From the perspective of this file we need a few more ../ to get the relative path right. I'll think of a nicer path to use, but want to do that when changing this header on all files later.